src/edu/psu/sweng500/team8/coreDataStructures/Board.java

```diff
@@ -10,8 +11,10 @@
  * Data holder and controller for a Sudoku grid Used to validate game
  * constraints
  */
-public class Board {
+public class Board implements Serializable {
```

Comment:
I'd like to figure out why BinarySerializable didn't work for you...we may need to refactor this